### PR TITLE
Update Builder Extensions guide

### PR DESCRIPTION
**Pull request by anuruddhal**
## Purpose
- Update Builder Extensions guide with new syntaxes and instructions

### CODE REVIEW

File: website/two-column-pages/docs/learn/how-to-extend-ballerina.md [hunk 1]
```diff
@@ -445,15 +457,14 @@ In the maven project, create a hello-extension/src/main/ballerina/ballerinax/doc
...
+documentation {Hello annotation configuration
+    F{{salutation}} - Greeting
+}
 public type HelloConfiguration {
-   string salutation;
+    string salutation;
 };
 
-@Description {value:"THIS TEXT APPEARS IN API DOCS"}
+documentation {@hello:Greeting annotation configuration}
```
Comment (hasithaa) [inline website/two-column-pages/docs/learn/how-to-extend-ballerina.md]: Is this documentation correct? what is `@hello` means here. ?

File: website/two-column-pages/docs/learn/how-to-extend-ballerina.md [hunk 2]
```diff
@@ -599,6 +610,164 @@ Configure Maven compiler plugin. Ballerina requires Java8 for the builder extens
...
+            <includes>
+                <include>**</include>
+            </includes>
+        </fileSet>
+    </fileSets>
+</assembly>
+```
+
+Create a file named `Ballerina.toml` in  `hello-extension/src/main/ballerina/ballerinax` folder and add following content.
+```ballerina
```
Comment (hasithaa) [inline website/two-column-pages/docs/learn/how-to-extend-ballerina.md]: This shouldn't ballerina. Should be toml.

File: website/two-column-pages/docs/learn/how-to-extend-ballerina.md [hunk 3]
```diff
@@ -611,6 +780,9 @@ The resulting `target/hello-extension-1.0-SNAPSHOT.jar` will have the annotation
 
 Place the jar file at `<ballerina_tools_home>/bre/lib` of your Ballerina distribution.
 
+Extract `target/hello-extension-0.970.5-ballerina-binary-repo.zip` file and copy the `repo/ballerinax` folder to `<ballerina_tools_home>/repo/` folder.
```
Comment (hasithaa) [inline website/two-column-pages/docs/learn/how-to-extend-ballerina.md]: The version should be 0.975.0

File: website/two-column-pages/docs/learn/how-to-extend-ballerina.md (hunk 1 above)
Comment (anuruddhal) [reply website/two-column-pages/docs/learn/how-to-extend-ballerina.md]: @hello:Greeting{} is the annotation.

**Pull request merged**